README.md

```diff
@@ -19,31 +19,33 @@ $ pip install -I pelion_test_lib*.whl
 
 ## Basic usage
 
-- Build the [Device Management Client example application](https://www.pelion.com/docs/device-management/current/connecting/device-management-client-tutorials.html) for your board and flash it.
-- The [API key](https://www.pelion.com/docs/device-management/current/integrate-web-app/api-keys.html) is used from Mbed's default `CLOUD_SDK_API_KEY` environment variable, but you can override it by defining a separate variable using a command-line command:
-    - Linux: `export PELION_CLOUD_API_KEY=[api_key_here]`
-    - Windows: `set PELION_CLOUD_API_KEY=[api_key_here]`
+- Build the [Device Management Client example application](https://developer.pelion.com/docs/device-management/current/connecting/mbed-os.html) for your board and flash it.
+- Set `PELION_CLOUD_API_KEY` environment variable with your [access key](https://developer.pelion.com/docs/device-management/current/user-account/application-access-keys.html).
```

Comment:
We probably could deprecate these API_KEY macros and only discuss about access keys.

API keys are deprecated.

Reply:
this requires changes in the library which I prefer not to handle right now.

Reply:
But I think this can be seperate PR anyways.

Reply:
Let's leave that for SyTE to decide and handle.